kolibri/core/assets/src/views/SideNav.vue

```diff
@@ -51,6 +52,8 @@
               <p :style="{ color: $themeTokens.annotation, fontSize: '12px', marginTop: 0 }">
                 {{ getUserKind }}
               </p>
+
+              <SideNavStorageNotification />
               <!-- display sync status, when relevant -->
               <div v-if="isSubsetOfUsersDevice" data-test="syncStatusInDropdown">
                 <div class="sync-status">
```

Comment:
What you've done here is add a "storage notification" in addition to the existing "sync status" notification. That makes sense from a code perspective, because as we discussed with @bjester and @rtibbles, they are different types of syncs. 

However, in the current designs, we already have a "Device status" header and in the figma, these two types of statuses are displayed under "Device status". They're... conflated, for lack of a better term. 

From the user perspective, having the two different statuses might be confusing - "why are there two different types of syncs?" However, I'm not sure if there might be any edge cases that would warrant it. Perhaps @bjester could weigh in, based on his work on the backend that supports these notifications for now. 

My initial thought is that _in the sidenav_, we add this additional string for the insufficient storage to the existing sync status display. If there is insufficient storage, display that string (no matter the other successes or failures of the facility data syncing. Otherwise, display the existing sync status, whatever it may be. 

What do you think? Blaine, Richard, @jtamiace - any thoughts?

Reply:
Thanks for looping me in, @marcellamaki. The API endpoint that returns the value passed to prop `userSyncStatus` on this component, which is then passed to `SyncStatusDisplay` below, will have a new status string `INSUFFICIENT_STORAGE`. We can only know the device status if it synced successfully, so the error states are mutually exclusive.